veracruz-server/Cargo.toml

```diff
@@ -63,6 +64,7 @@ nix = { version = "0.20.2", optional = true }
 postcard = "0.7.2"
 policy-utils = { path = "../policy-utils", optional = true }
 psa-attestation = { path = "../psa-attestation", optional = true }
+rand = "0.7"
```

Comment:
Could you use `rand = "0.8.3"` (used in the rest of Veracruz) to avoid duplication?

Reply:
Done.